website/util/rubeus.py

```diff
@@ -116,6 +118,11 @@ def build_addon_root(node_settings, name, permissions=None,
         'nodeApiUrl': node_settings.owner.api_url,
     }
     ret.update(kwargs)
+
+    if flag_is_active(request, 'storage_usage_feature'):
```

Comment:
This won't quite work because you're passing a Flask request and `waffle.flag_is_active`. See `website/ember_osf_web/decorators.py` for the workaround we did for storage_i18n.